### DISCUSSION TITLE
Which Open Source Observability / Monitoring Solution Should we Support?

### DISCUSSION THREAD

**Discussion by pmarkowsky** [google/santa#681] [Q&A]
Which monitoring/observability solutions are people using / would you like Santa to integrate with?

**Comment by np5** [comment]
Sorry for the late reply!

[OpenMetrics](https://github.com/OpenObservability/OpenMetrics) (a superset of the Prometheus exposition format) seems to be the current standard, but it still requires an extra agent on the endpoints to collect the metrics.

What about sending a summary of the metrics in the preflight or postflight phase of the sync? A lot of the granularity would be lost, but it would still offer better observability without requiring another agent.
